### DISCUSSION TITLE
LookDown: Ability to clear values

### DISCUSSION THREAD

**Discussion by wenstarner** [khoait/DCE.PCF#144] [Ideas] [answered]
Hi, just an idea that came to mind when doing some testing on a form...

There doesn't appear to be any ability to clear values, once one has been selected.

It's just an idea, but perhaps the selected value could have a little 'x', like the OOB lookup component?

For example, below is a screenshot with LookDown on the top, and the OOB component on the bottom.

![image](https://github.com/user-attachments/assets/fce7649e-8651-4a4a-a87c-7364a5bf00af)

**Comment by khoait** [comment, marked as answer]
@wenstarner thanks for your idea! I'll include this in the next release of the control. Cheers!
